android/app/src/main/java/org/fossasia/openevent/viewmodels/FeedFragmentViewModel.java

```diff
@@ -0,0 +1,35 @@
+package org.fossasia.openevent.viewmodels;
+
+import android.arch.lifecycle.LiveData;
+import android.arch.lifecycle.MutableLiveData;
+import android.arch.lifecycle.ViewModel;
+
+import org.fossasia.openevent.data.facebook.FeedItem;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FeedFragmentViewModel extends ViewModel {
+
+    private MutableLiveData<List<FeedItem>> feedLive;
+
+    public FeedFragmentViewModel() {
+        feedLive = new MutableLiveData<>();
+    }
+
+    public LiveData<List<FeedItem>> getFeedList() {
+        if (feedLive == null)
```

Comment:
Isn't this null check redundant ? Since we're already initialising it in the constructor, how can it be null after that ?